app/interactors/tasks/create/send_notification.rb

```diff
@@ -0,0 +1,24 @@
+module Tasks
+  class Create
+    class SendNotification
+      include Interactor
+
+      delegate :project, :task, to: :context
+
+      def call
+        mail_owner.deliver_later
+        mail_members.deliver_later
+      end
+
+      private
+
+      def mail_owner
+        @mail_owner ||= TaskMailer.task_created_to_owner(project, task)
+      end
+
+      def mail_members
+        @mail_members ||= TaskMailer.task_created_to_members(project, task)
```

Comment:
I think additional methods for these mails are unnecessary. You can put them in the `call` method.